.gitignore

```diff
@@ -3,3 +3,4 @@ npm-debug.*
 test/fixtures/(*
 test/fixtures/shebang_noenv
 test/tmp
+.vscode/
```

Comment:
This should probably be in your [global gitignore](https://help.github.com/articles/ignoring-files/).